docs/content/usage/actions/act-runner.en-us.md

```diff
@@ -262,6 +262,40 @@ The runner will fetch jobs from the Gitea instance and run them automatically.
 
 Since act runner is still in development, it is recommended to check the latest version and upgrade it regularly.
 
+## Systemd service
+
+It is also possible to run act-runner as a [systemd](https://en.wikipedia.org/wiki/Systemd) service. Create an unprivileged `act-runner` user on your system, and the following file in `/etc/systemd/system/act-runner.service`. The paths in `ExecStart` and `WorkingDirectory` may need to be adjusted depending on where you installed the `act-runner` binary, its configuration file, and the home directory of the `act-runner` user.
+
+```ini
+[Unit]
+Description=Gitea Actions runner
+Documentation=https://gitea.com/gitea/act_runner
+After=docker.service
+
+[Service]
+ExecStart=/usr/local/bin/act-runner daemon --config /etc/act-runner/config.yaml
+ExecReload=/bin/kill -s HUP $MAINPID
+WorkingDirectory=/var/lib/act-runner
```

Comment:
Is a working directory a requirement? If not, I would recommend running in `/tmp`.

Reply:
Working directory is required due to the .runner file.